packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -133,274 +125,33 @@ export default class VTable {
      * @param format Table format to apply
      */
     applyFormat(format: Partial<TableFormat>) {
-        if (!this.table) {
+        if (!this.table || !format) {

```

Comment:
what if format is incomplete? We can keep the default format here and use it when applyFormat is called.

applyFormat() can pass in a partial format object, then we need to combine it with default format and existing format to make sure it has a complete value:

```ts
format = { ...defaultFormat, ...(this.formatInfo || {}), ...(format || {}) };
```

In this way it is guaranteed existing format (if any) is respected and new format is complete